java-notebooks/google-cloud-notebooks/src/test/java/com/google/cloud/notebooks/v1beta1/it/ITNotebookServiceClientTest.java

```diff
@@ -66,6 +66,8 @@ public class ITNotebookServiceClientTest {
   public static void setUp() throws IOException, ExecutionException, InterruptedException {
     // Create Test Notebook Instance
     client = NotebookServiceClient.create();
+    Util.cleanUpNotebookInstances(client, PARENT);
```

Comment:
Wouldn't this delete the one created above before it's even used?
Also, why isn't cleanup in `tearDown()` sufficient?

Reply:
It would only delete notebooks that have a creation time more than 24 hours earlier.

Reply:
Can we have a prefix for the notebook names or something to avoid accidentally deleting unrelated notebooks.

Reply:
Yep, it's similar to the comment above about compute and container. I'll add in more specific names for the IT instances.